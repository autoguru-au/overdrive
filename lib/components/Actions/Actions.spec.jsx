import React from 'react';
import { render } from '@testing-library/react';
import { Actions } from '.';

describe('<Actions />', () => {
	it('should not throw', () => {
		expect(() => render(<Actions />)).not.toThrow();
	});

	it('should match the snapshot', () => {
		expect(render(<Actions />).container.firstChild).toMatchSnapshot();
	});

	it('should pass className down', () => {
		expect(
			render(<Actions className="test" />).container.firstChild,
		).toHaveClass('test');
	});
});
