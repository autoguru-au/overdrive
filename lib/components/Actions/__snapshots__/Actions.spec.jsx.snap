// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Actions /> should match the snapshot 1`] = `
<div
  class="root equalWidth"
/>
`;
